src/Components/Common/Input/Input.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {ChangeEvent} from "react";
+import styled from "styled-components";
+
+const StyledInput = styled.input`
+  color: #c2c0c0;
+  background-color: #2f2f2f;
+  border-radius: 2px;
+`;
+
+interface Props {
+    type: string;
+    placeholder: string;
+    width: string;
+    height: string;
+    handleInputChange: (e: ChangeEvent<HTMLInputElement>) => void;
+}
+
+export const Input = (props: Props) => (
```

Comment:
To jest dobre miejsce na zastosowanie destrukturyzacji. Nie będzie trzeba w dalszej części używać prefixu props. Dodatkowo np. w atrybucie style mógłbyś zastosować zapis skrócony style={{width, height}}

Reply:
Ok.